src/app/main/component/user/components/profile/user-notifications/user-notifications.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="container" aria-label="user-notification">
+  <h1 class="heading h5">{{ 'homepage.notifications.title' | translate }}</h1>
+  <div *ngIf="notifications.length" class="filter-search-wrapper d-flex">
+    <div class="filter-wrapper d-flex">
+      <span class="filter-header h6">Filter by</span>
+      <div class="filter-button">
+        <button class="global-tag">All</button>
+        <button *ngFor="let filter of projects" class="tag-button">
+          <a class="custom-chip global-tag" [ngClass]="{ 'global-tag-clicked': true }">
```

Comment:
If you're setting the global-tag-clicked class to true, there's no need to use a binding at all. Just `<a class="custom-chip global-tag global-tag-clicked">`

Reply:
Of course. It was added to see how it looks. I removed this class for now. I'll  implement it  when I add a filter